### PR TITLE
javascript-local-storage - Bryan Brown

### PR DESCRIPTION
**Pull request by bbrown949**
Screenshot 2023-04-07 at 2 40 16 PM

### CODE REVIEW

File: javascript-local-storage/data.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/* exported todos */
+
+let todos = [];
+
+const previousTodosJSON = localStorage.getItem('javascript-local-storage');
+
+todos = JSON.parse(previousTodosJSON);
```
Comment (shawnkost) [inline javascript-local-storage/data.js]: You should have an `if` statement checking if `previousTodosJSON` is null

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Bryan!

File: javascript-local-storage/data.js (hunk 1 above)
Comment (bbrown949) [reply javascript-local-storage/data.js]: updated.

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**